### PR TITLE
74 l make the pdf in html

### PR DESCRIPTION
**Pull request by tomltd**

### CODE REVIEW

**Comment by tomltd** [comment]
Wow this is great now @katjam

File: src/assessment.handlebars [hunk 1]
```diff
@@ -622,13 +622,403 @@
...
+      <ul aria-label="menu">
+        <li>
+          <button class="nav-link pick" data-action="start-or-resume">Try some questions</button>
+        </li>
+        <li>
+          <button class="nav-link break" data-action="break">Have a break</button>
+        </li>
+        <li>
+          <button class="nav-link break" data-action="about-pip">Read more about PIP</button>
+        </li>
```
Comment (katjam) [inline src/assessment.handlebars]: Change this one to 'Read less about PIP' ... @harryharrold copy for button getting back to the Essential guide from the html'd pdf.

See http://esa-assessment.support/#guide for the live example of same thing on ESA which we also need to change.

**Comment by katjam** [comment]
:sunrise: Basically there but I'd like to change the button copy I commented on.

**Comment by tomltd** [comment]
@katjam I'm confused by this :confused: am I waiting copy from @harryharrold for the 'Read more about PIP' to indicate that is should take you to the about pip page?

**Comment by tomltd** [comment]
@harryharrold can we have some copy for this plzz :palm_tree:

**Comment by tomltd** [comment]
@katjam REEADY

**Comment by katjam** [comment]
yup :neckbeard:

**Pull request merged**